core/domain/collection_domain.py

```diff
@@ -129,6 +129,31 @@ def __init__(
             commit_type, commit_message, commit_cmds, version,
             post_commit_status, post_commit_community_owned,
             post_commit_is_private):
+        """Constructs an CollectionCommitLogEntry domain object.
+
+        Args:
+            created_on: datetime.datetime. Date and time when the collection
+                commits were created.
+            last_updated: datetime.datetime. Date and time when the collection
+                commits were lastly updated.
+            user_id: str. User id of the user.
+            username: str.  Username of the user who has made the commit.
+            collection_id: str. Id of the collection.
+            commit_type: str.
+            commit_message: str. A description of changes made to the
+                collection.
+            commit_cmds: list(dict). A list of change commands made to the given
+                collection.
+            version: int. The version of the collection.
+            post_commit_status: str. The status of the post commit made to the
+                collection.
+            post_commit_community_owned: bool. A boolean value true if the post
```

Comment:
Ditto, what is a "post commit"?

(FYI, "post" here actually means "after".)

Also, for boolean values, you can do something like: "bool. Whether the ..."